zamba/data/video.py

```diff
@@ -325,6 +325,41 @@ def validate_total_frames(cls, values):
         return values
 
 
+def get_cached_array_path(config, vid_path, cache_dir):
+    """Get the path to where the cached array would be, if it exists.
+
+    config: dictionary or VideoLoaderConfig
+    vid_path: string path to the video, or Path, or S3Path
+    cache_dir: string directory name, or Path, or S3Path
+
+    returns: Path object to the cached data
+    """
+    if isinstance(config, VideoLoaderConfig):
+        # NOTE: config.dict() and dict(config) are equal, but their string representations are not
+        # so we should use config.dict() everywhere
+        config = config.dict()
+
+    # don't include `cleanup_cache` or `cache_dir` in the hashed config
+    # NOTE: sorting the keys avoids a cache miss if we see the same config in a different order
+    keys = config.keys() - {"cleanup_cache", "cache_dir"}
+    hashed_part = {k: config[k] for k in sorted(keys)}
+    print(hashed_part)
```

Comment:
this should be logged instead of printed if we intend to keep it, but i think this is a leftover from development and can be removed

Reply:
Removed.